gridData/OpenDX.py

```diff
@@ -459,6 +472,13 @@ def __init__(self,classid='0',components=None,comments=None):
         self.components = components
         self.comments= comments
 
+    def _openfile(self, filename):
+        """Returns a regular or gz file descriptor"""
```

Comment:
Say in docs that the file is opened for writing.

Reply:
done!